Opentelemetry OCI guide (#1017)
micronaut-projects/micronaut-guides: guides/micronaut-cloud-trace-base/start.adoc

common:header.adoc[]

> This guide you will discover how simple it is to add Tracing to a  Micronaut application.

Reviewer: ```suggestion
In this guide, you'll discover how simple it is to add tracing to a Micronaut application.
```

> Tracing allows you to track service requests in a single application or a distributed one. Trace data shows the path, time spent in each section (called a span) and other data collected along the way. Tracing gives you observability into what is causing bottlenecks and failures.

Reviewer: ```suggestion
Tracing allows you to track service requests in a single application or a distributed one. Trace data shows the path, time spent in each section (called a span), and other information collected along the way. Tracing gives you observability into what is causing bottlenecks and failures.
```
